xml/System.Security.Permissions/KeyContainerPermissionFlags.xml

```diff
@@ -138,15 +138,7 @@
       </ReturnValue>
```

Comment:
not related to your changes but the remarks for this type has a lit of the powerful flags that should be fenced to prevent localization and that paragraph be added inside the caution note